vscode/src/test/testutils.ts

```diff
@@ -333,42 +333,298 @@ export function runTestSuite(folder: string): Promise<void> {
 	});
 }
 
-export const awaitClient = async () : Promise<NbLanguageClient> => {
-    const extension = vscode.extensions.getExtension(extConstants.ORACLE_VSCODE_EXTENSION_ID);
-    if (!extension) {
-        return Promise.reject(new Error(l10n.value("jdk.extension.notInstalled.label")));
-    }
-    if(extension.isActive){
-        return globalVars.clientPromise.client;
-    }
-    const waitForExtenstionActivation : Thenable<NbLanguageClient> = extension.activate().then(async () => {
-        return await globalVars.clientPromise.client;
-    });
-    return Promise.resolve(waitForExtenstionActivation);
+export const awaitClient = async (): Promise<NbLanguageClient> => {
+	const extension = vscode.extensions.getExtension(extConstants.ORACLE_VSCODE_EXTENSION_ID);
+	if (!extension) {
+		return Promise.reject(new Error(l10n.value("jdk.extension.notInstalled.label")));
+	}
+	if (extension.isActive) {
+		return globalVars.clientPromise.client;
+	}
+	const waitForExtenstionActivation: Thenable<NbLanguageClient> = extension.activate().then(async () => {
+		return await globalVars.clientPromise.client;
+	});
+	return Promise.resolve(waitForExtenstionActivation);
+}
+
+export function findClusters(myPath: string): string[] {
```

Comment:
Is this method still required to be added to *testutils.ts*?

Reply:
Seems like it's not being used anywhere post the refactoring . I am actually not adding this one the diff is  a bit misleading . Nevertheless I feel we can keep this even though we are not using it since we may intend to use it in the future .